types/test/isMap.ts

```diff
@@ -0,0 +1,17 @@
+import * as RA from 'ramda-adjunct';
+
+RA.isSet(new Map()); // $ExpectType boolean
```

Comment:
Tests here are testing `isSet` instead of `isMap`.

Reply:
Oops; forgot to change that, sorry! I've updated it